### PR TITLE
Added all periods to muon analysis 

### LINKED ISSUE #19704: Add all periods to multifit
Users want an add all periods option (as default) for the multiple fitting mode in muon analysis.

### PR DESCRIPTION
**Pull request by AnthonyLim23**
Added all periods option to the muon analysis multiple fitting GUI. 

In muon analysis load some multiperiod data (HIFI 84447 is good) and make sure multiple fitting mode is enabled in the settings tab. 

The period selection should default to all periods.

If you press the combine button at the bottom, then enter some numbers into the popup, it should be added to the list of selected periods. If you change selected periods and change back to all the display should update.  
 
Fixes #19704 .

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [x] Is the code of an acceptable quality?
- [x] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [x] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: MantidQt/MantidWidgets/src/MuonFitPropertyBrowser.cpp [hunk 1]
```diff
@@ -168,7 +168,8 @@ void MuonFitPropertyBrowser::init() {
   tmp = "bwd";
   addGroupCheckbox(tmp);
   m_periodsToFit = m_enumManager->addProperty("Periods to fit");
-  m_periodsToFitOptions << "1"
+  m_periodsToFitOptions << "All Periods"
```
Comment (AntonPiccardoSelg) [inline MantidQt/MantidWidgets/src/MuonFitPropertyBrowser.cpp]: If you are using `All Periods` and the other values only in `MuonFitPropertyBrowser` then it is worth defining them as variables in a `static const`. This way we write the string only once.

**Review by AntonPiccardoSelg — CHANGES_REQUESTED** [review]

File: MantidQt/MantidWidgets/src/MuonFitPropertyBrowser.cpp (hunk 1 above)
Comment (AnthonyLim23) [inline MantidQt/MantidWidgets/src/MuonFitPropertyBrowser.cpp]: Is that still true as we update the string within MuonFitPropertyBrowser?

**Review by AntonPiccardoSelg — APPROVED** [review]

File: MantidQt/MantidWidgets/src/MuonFitPropertyBrowser.cpp (hunk 1 above)
Comment (AnthonyLim23) [inline MantidQt/MantidWidgets/src/MuonFitPropertyBrowser.cpp]: I will fix these in #19713

**Comment by AntonPiccardoSelg** [comment]
Works well, please :shipit:

**Pull request merged**